server/views/outcomes/resultedCases.njk

```diff
@@ -1,4 +1,5 @@
 {%- set moment = params.moment -%}
+{%- set moment = params.moment -%}
```

Comment:
Is that needed twice?

Reply:
Yea, plan B  😉 
accidental  CMD+D keys
will remove